### DISCUSSION TITLE
Post Links to created/updated PRs as notice

### DISCUSSION THREAD

**Discussion by Nef10** [BetaHuhn/repo-file-sync-action#100] [Ideas] [answered]
I often like to see/open the created PRs after the action ran.

Actions can post annotation from their run, which will appear on the summary page, e.g. like when we have an error:

<img width="2647" alt="Screen Shot 2021-09-04 at 11 19 31 PM" src="https://user-images.githubusercontent.com/4551135/132117568-19c40fc8-2cc7-4d49-8a68-ecacbd08be81.png">

Next to posting errors, actions can also post warnings and notices there, see [here](https://github.com/actions/toolkit/tree/main/packages/core#annotations). 

I think it would be handy to put the `Pull Request #X created/updated: https://github.com/Y/Z/pull/X` output as notice there. What to you think?

**Comment by BetaHuhn** [comment]
I wanted to do something similar a while back with the [actions parameter](https://docs.github.com/en/rest/reference/checks#create-a-check-run--parameters). It shows an actual button to click in the summary page. The problem was that you can't just define a handler for the click and instead have to listen to a webhook being sent to a GitHub app which doesn't work in our case. 

Does GitHub linkify the URL in a notice? Either way, your solution with the notice seems to be a good compromise 👍🏼

**Comment by Nef10** [comment]
Yes, it does link them, see [here](https://github.com/Nef10/sync-test/actions/runs/1204289659).

**Comment by BetaHuhn** [comment, marked as answer]
:tada: This a part of version [v1.13.0](https://github.com/BetaHuhn/repo-file-sync-action/releases/tag/v1.13.0) :tada:

To use the latest version, use the Action with the `v1` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@v1
```

Or the `latest` tag:

```yml
uses: BetaHuhn/repo-file-sync-action@latest
```
